pkg/flag/options.go

```diff
@@ -202,9 +202,9 @@ func (f *Flags) groups() []FlagGroup {
 	if f.SecretFlagGroup != nil {
 		groups = append(groups, f.SecretFlagGroup)
 	}
-	if f.LicenseFlagGroup != nil {
-		groups = append(groups, f.LicenseFlagGroup)
-	}
+	//if f.LicenseFlagGroup != nil {
+	//	groups = append(groups, f.LicenseFlagGroup)
+	//}
```

Comment:
why is this block commented?

Reply:
Oh... thank you! I checked another issue.
I removed it.